content/articles/understanding-how-cluster-marker-works/index.md

```diff
@@ -0,0 +1,240 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+ This is no fact again as we are all aware that BIG DATA is the keyword of the 21st century, and it's shocking to know that only a few people understand that a large part of data is categorized as geospatial. Nowadays nearly all of the gadgets we use are satellite-based systems like GPS. Today, the simplest case of displaying data/marker on a map with an acceptable user experience may be a real challenge especially for the user, they might find it difficult to understand or to analyze data at a particular zoom level. For instance, a platform that has users with hundreds of markers on the map. When you zoom out the map, these markers all overlap and make the map appear messy and crowded. To improve this kind of map, it is professional to use the marker clustering method, and this is the most efficient technique of grouping markers together.
+
+### Prerequisites
+To Understand this tutorial it is assumed that the reader knows the below technologies:
+- JavaScript
+- HTML/CSS
+### Goal
+
+At the end of this guide, we'll build a map with clustering features, and these features include:
+- Leaflet map and its usage.
+- Leaflet Marker Cluster with beautiful animation.
+### Getting Started
+
+Have you ever tried to visualize a large amount of data on the map? If so, using the normal method you will end up like this, lots of markers here and there.
+
+![image info](engineering-education/understanding-how-cluster-marker-works/marker.png)
+
+Instead of visualizing a map using the normal method, it is important to consider the usage of the Clustering method which is the most efficient way of mapping when it comes to visualizing big data. To achieve the clustering of markers, the usage of the leaflet-marker-cluster plugin is needed, So let get started. we need to initialize our map for data visualization, open any code editor and create an `index.html` file in a new project directory, then copy and paste snippet the below.
```

Comment:
Eliminate the highlighted portion - it is important to consider `the usage of` the Clustering method